### PR TITLE
Fix:resolve cannot open electron window automatically after update node version to v18

### PR DESCRIPTION
**Pull request by JeffreyMa597**
What problem does this PR solve?
------
After updating our node version to v18, we got a problem that the neuron client cannot be opened automatically.

Ref: https://github.com/Magickbase/neuron-public-issues/issues/80

before: 
image

after:
image

Check List
------

### Test
e2e Test
### Task
none

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "bootstrap": "npx cross-env lerna bootstrap && lerna link",
     "start:ui": "cd packages/neuron-ui && yarn run start",
     "start:wallet": "cd packages/neuron-wallet && yarn run start:debug",
-    "start": "concurrently \"cross-env BROWSER=none yarn run start:ui\" \"wait-on http://localhost:3000 && yarn run start:wallet\"",
```
Comment (yanguoyu) [inline package.json:28]: localhost can't wait on?

**Comment by Keith-CY** [comment]
Please add a brief explanation if possible

**Comment by JeffreyMa597** [comment]
> Please add a brief explanation if possible

sure.

**Comment by JeffreyMa597** [comment]
This is the issue I found in the `wait-on` repo. 
https://github.com/jeffbski/wait-on/issues/109

**Review by yanguoyu — APPROVED** [review]

**Comment by JeffreyMa597** [comment]
@Cedar67 Could you please verified it on Linux and Windows?

**Review by Keith-CY — APPROVED** [review]

**Review by Cedar67 — APPROVED** [review]
Verified on Windows and Mac OS.

**Pull request merged**